src/Yasumi/ProviderInterface.php

```diff
@@ -34,7 +34,7 @@ public function initialize(): void;
      * Returns a list of sources (i.e. references to websites, books, scientific papers, etc.) that are
      * used for determining the calculation logic of the providers' holidays.
      *
-     * @return array<string> a list of external sources (empty when no sources are defined)
+     * @return list<string> a list of external sources (empty when no sources are defined)
```

Comment:
I am not familiar using 'list' as the return type. Are you using a particular tool that requires this to be 'list' instead of 'array'. Furthermore, this change causes a warning by Phan: 

`The inherited return type 'list<string>' for Yasumi\ProviderInterface::getSources is more specific than the implemented return type for Yasumi\Provider\Argentina::getsources 'array<array-key, string>' `

Reply:
Apologies for the late reply. I am just a bit reluctant to use tool specific syntax/conventions.

Reply:
I don't think that this syntax is specific for Psalm anymore. Also PHPStan and PhpStorm support this syntax.